controllers/article-controller.js

```diff
@@ -1,5 +1,16 @@
 const Article = require('../models/Article.js')
 
+exports.getAllArticles = async (req, res) => {
+  const {} = req.query
+
+  try {
+    const foundArticles = await Article.findMany()
+    return res.json({ articles: foundArticles })
```

Comment:
I don't believe the `return` is necessary to have, one small way this could be cleaned is if `foundArticles` is renamed to `articles` allowing you to return `{ articles }` as a shorthand